order/bas/readme.md

```diff
@@ -21,22 +21,24 @@ click 'Run Sub/User Form" and all break points should land on "Test Passed"
 
 Slide listener will listen for these tags and behave approprately.
 
```

Comment:
Why did you remove bulleted list?

Reply:
To better support a copy/paste workflow. I was afraid the bullet styling in the notes field when I would triple click--> copy-->paste would potentially affect the tag or it's detection (which it shouldn't, so really it was just because I was superstitious)